simulator/src/resim/cmd_set_current_epoch.rs

```diff
@@ -46,21 +46,22 @@ impl SetCurrentEpoch {
             },
             &blobs,
             DEFAULT_MAX_CALL_DEPTH,
-            &mut track,
+            track,
             &mut scrypto_interpreter,
             Vec::new(),
         );
 
         // Invoke the system
-        kernel
+        let output = kernel
             .invoke(EpochManagerSetEpochInvocation {
                 epoch: self.epoch,
                 receiver: EPOCH_MANAGER,
             })
             .map_err(Error::TransactionExecutionError)?;
 
+        let receipt = kernel.finalize(Ok(Vec::new()));
```

Comment:
Shoudn't we pass the output to `finalize`?

Reply:
not sure :sweat_smile: I thought it was done like that on purpose

Reply:
ah, that's because the invoke output is now typed and `finalize` specifically requires the output of `TransactionProcessorRunInvocation`